_data/projects/matrixLab.yml

```
@@ -0,0 +1,16 @@
+---
+name: matrixLab
+desc: matrixLab is a compilation of C++ libraries to carry out fundamental matrix operations in C++
+site: https://github.com/bvks2020sameer/matrixLab
+tags:
+- c++
+- matrix
+- mathematical-engine
+- cmake
+- msvc-2022
+- cross-platform
+- calculation
+- machine-learning
+upforgrabs:
+  name: enhancement
+  link: https://github.com/bvks2020sameer/matrixLab/issues?q=is%3Aissue+is%3Aopen+label%3Aenhancement
```

:art: a more succinct label

```suggestion
  link: https://github.com/bvks2020sameer/matrixLab/labels/enhancement
```